src/main/java/uk/gov/companieshouse/confirmationstatementapi/model/json/ConfirmationStatementSubmissionDataJson.java

```diff
@@ -0,0 +1,17 @@
+package uk.gov.companieshouse.confirmationstatementapi.model.json;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class ConfirmationStatementSubmissionDataJson {
```

Comment:
I'd leave off the 'Json' from the class name as Json is a string/text, 'ConfirmationStatementSubmissionData' would be the right name as it is a class representing that data.

Reply:
I'm just distinguishing it from the DAO if we name it like this suggestion it could get confusing.